### ISSUE TITLE
`module.exports.property` results in a property of type any

### ISSUE THREAD

**Issue by sandersn** [Microsoft/TypeScript#25621]
```js
module.exports.y = "hi"
let u = module.exports.y/**/
```

Get quick info at position.

**Expected behavior:**

The type of `module.exports.y` is reported as string.

**Actual behavior:**

The type of `module.exports.y` is reported as any, and there aren't any completions, so it seems like it actually is any.

Edit:
1. Repros on 2.9.2 and master.
2. When imported, the type is correct

```js
var { y } = require('./mod')
y/**/
```

The declared type is used if there is one, even if it contradicts the initialiser (eg `/** @type {number} */ module.exports.y = 'hi'`).
